examples/with-thirdweb/package.json

```diff
@@ -0,0 +1,26 @@
+{
+  "name": "next-typescript-starter",
+  "version": "0.1.0",
```

Comment:
```suggestion
```
These fields should be omitted for examples which is why this check is currently failing https://github.com/vercel/next.js/runs/6273678404?check_suite_focus=true